lookout/style/format/tests/test_quality_report.py

```diff
@@ -76,9 +76,11 @@ def __enter__(self):
         return self
 
     def __exit__(self, *args):
-        self.extend(self._stringio.getvalue().splitlines())
+        value = self._stringio.getvalue()
+        self.extend(value.splitlines())
         del self._stringio    # free up some memory
         sys.stdout = self._stdout
+        print(value)
```

Comment:
why do you change `__exit__()`?

Reply:
not printing the value on exit makes debugging very annoying when the test fails